src/main.c

```diff
@@ -102,9 +102,6 @@ int main (int argc, char* argv[])
 	close_telnet_socket();
 	close_unix_socket();
 
-	// Free regex list and array of whitelisted domains
-	free_regex();
```

Comment:
Shouldn't this be kept?

Reply:
The scope of the `free_regex()` routine has been reduced to the respective module. At this point, `pihole-FTL` will (unstoppable) terminate. This will free all allocated memory automatically.